_1327/documents/__init__.py

```diff
@@ -0,0 +1,2 @@
+# don't ask me why we need that... without it django will run into an error
```

Comment:
i will definitely not accept such comments in the code. please either look up why we need it or at least write "needed to avoid error ..."